examples/angular/src/app/components/content/content.component.ts

```diff
@@ -165,45 +172,95 @@ export class ContentComponent implements OnInit, OnDestroy {
     }
   }
 
+  async verifyMessage(
+    message: SignMessageParams,
+    signedMessage: SignedMessage
+  ) {
+    const verifiedSignature = verifySignature({
+      message: message.message,
+      nonce: message.nonce,
+      recipient: message.recipient,
+      publicKey: signedMessage.publicKey,
+      signature: signedMessage.signature,
+      callbackUrl: message.callbackUrl,
+    });
+    const verifiedFullKeyBelongsToUser = await verifyFullKeyBelongsToUser({
+      publicKey: signedMessage.publicKey,
+      accountId: signedMessage.accountId,
+      network: this.selector.options.network,
+    });
+
+    const isMessageVerified = verifiedFullKeyBelongsToUser && verifiedSignature;
+
+    const alertMessage = isMessageVerified
+      ? "Successfully verified"
+      : "Failed to verify";
+
+    alert(
+      `${alertMessage} signed message: '${
+        message.message
+      }': \n ${JSON.stringify(signedMessage)}`
+    );
+  }
+
+  verifyMessageBrowserWallet() {
+    const urlParams = new URLSearchParams(
+      window.location.hash.substring(1) // skip the first char (#)
+    );
+
+    const accId = urlParams.get("accountId") as string;
+    const publicKey = urlParams.get("publicKey") as string;
+    const signature = urlParams.get("signature") as string;
+
+    if (!accId && !publicKey && !signature) {
+      return;
+    }
+
+    const message: SignMessageParams = JSON.parse(
+      localStorage.getItem("message") as string
+    );
+
+    const signedMessage = {
+      accountId: accId,
+      publicKey,
+      signature,
+    };
+
+    this.verifyMessage(message, signedMessage);
```

Comment:
Add `await` for `this.verifyMessage` after making `verifyMessageBrowserWallet` async.